packages/frontend/src/Home.jsx

```diff
@@ -76,28 +74,26 @@ export default observer(() => {
                 onChange={(event) => { setNonce(event.target.value ?? 0) } }
               ></input>
               <p>current epoch key nonce: {nonce}</p>
-              
+
               <Button onClick={()=> {
-                userContext.requestReputation(posRep, negRep, graffitiPreimage, nonce)
-                setTotalPosRep(Number(posRep)+ Number(totalPosRep))
-                setTotalNegRep(Number(negRep)+ Number(totalNegRep))
```

Comment:
Here I mean to show the current epoch reputation (but not received)
and after transition, the user can check if all reputation are received in the new epoch

so current rep = current epoch rep (not received) + previous epoch rep (received)
the rep which is received is shown in 

https://github.com/Unirep/create-unirep-app/blob/9a677b4000897cd04772e4d5d2966de1d66af3e6/packages/frontend/src/Home.jsx#L43

Reply:
or we need a better UI to let users know the rep what they request in the epoch will be processed next epoch.